README.md

```diff
@@ -182,40 +127,13 @@ For more information on the registry and how to integrate with Teleporter dApps,
 
 ## Deploy Teleporter to a Subnet
 
-From the root of the repo, the TeleporterMessenger contract can be deployed by calling
```

Comment:
We should probably keep this documentation since we are keeping the `deploy_teleporter.sh` and `deploy_registry.sh` scripts.